docs/apache-airflow/upgrading-to-2.rst

```diff
@@ -0,0 +1,1164 @@
+                ],
+                "volume_mounts": [
+                    {
+                        "mountPath": "/foo/",
+                        "name": "example-kubernetes-test-volume",
+                    },
+                ]
+            }
+        }
+    )
+
+In the new model a user can accomplish the same thing using the following code under the ``pod_override`` key:
+
+.. code-block:: python
+
+    from kubernetes.client import models as k8s
+
+    second_task = PythonOperator(
+        task_id="four_task",
+        python_callable=test_volume_mount,
+        executor_config={"pod_override": k8s.V1Pod(
+            spec=k8s.V1PodSpec(
+                containers=[
+                    k8s.V1Container(
+                        name="base",
+                        volume_mounts=[
+                            k8s.V1VolumeMount(
+                                mount_path="/foo/",
+                                name="example-kubernetes-test-volume"
+                            )
+                        ]
+                    )
+                ],
+                volumes=[
+                    k8s.V1Volume(
+                        name="example-kubernetes-test-volume",
+                        host_path=k8s.V1HostPathVolumeSource(
+                            path="/tmp/"
+                        )
+                    )
+                ]
+            )
+        )
+        }
+    )
+
+For Airflow 2.0, the traditional ``executor_config`` will continue operation with a deprecation warning,
+but will be removed in a future version.
+
+
+Step 7: Upgrade to Airflow 2.0
+'''''''''''''''''''''''''''''''
+
+After running the upgrade checks as described above, installing the backported providers, modifying
+the DAGs to be compatible, and updating the configuration settings, you should be ready to upgrade to Airflow 2.0.
+
+A final run of the upgrade checks is always a good idea to make sure you have missed anything. At
+this stage the problems detected should be either be zero or minimal which you plan to fix after
+upgrading the Airflow version.
+
+At this point, just follow the standard Airflow version upgrade process:
+* Make sure your Airflow meta database is backed up
+* Pause all the DAGs and make sure there is nothing actively running
+* Upgrade the Airflow version to the 2.0 version of choice
+* Upgrade the Airflow meta database using ``airflow db upgrade``. This command is now shown in the Airflow 2.0 CLI syntax.
+* Restart Airflow Scheduler, Webserver, and Workers
+
+.. note::
+
+    The database upgrade may take a while depending on the number of DAGs in the database and the volume of history
+    stored in the database for task history, xcom variables, etc.
+    For a faster database upgrade and for better overall performance, it is recommended that you periodically archive
+    the old historical elements which are no longer of value.
+
+
+
+Appendix
+''''''''
+
+Changed Parameters for the KubernetesPodOperator
```

Comment:
These headings are at the same level -- they probably shouldn't be.

Reply:
Changed now to be sub-headings.